File: practice/java/Pattern_S.java
import java.util.*;
public class test
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner (System.in);
        System.out.println("Enter the range : ");
        int L = sc.nextInt();
        for(int i=0; i<L; i++)
        {
            System.out.print("* ");
        }
        System.out.println();
        for(int i=0; i<L; i++)
        {
            System.out.println("*");
        }
        for(int i=0; i<L; i++)
        {
            System.out.print("* ");
        }
        System.out.println();
        for(int i=0; i<L; i++)
        {
            for(int j=0; j<L; j++)
            {
                if(j==L-1)
                {
                    System.out.print("*");
                }
                else
                {
                    System.out.print("  ");
                }
            }
            System.out.println();
        }
        for(int i=0; i<L; i++)
        {
            System.out.print("* ");
        }
    }
}

File: practice/python/pattern_S.py
L=int(input("Enter the range : "));
for i in range(L):
    print("* ",end="");
print()
for i in range(L):
    print("*")
for i in range(L):
    print("* ",end="")
print()
for i in range(L):
    for j in range(L):
        if(j==L-1):
            print("*",end="");
        else:
            print("  ",end="");
    print();
for i in range(L):
    print("* ",end="")
    